Added question to FAQ on delays discrepancy

rdowinton opened the pull request:
Added question to FAQ on how request speed differs when using Crawlera as opposed to routing requests through proxies directly.

@qrilka can you please review? Thanks

qrilka commented on crawlera.rst:
```diff
@@ -453,3 +453,10 @@ Where can I monitor my Crawlera usage?
+
+If you're using your own proxies, you may notice a discrepancy in speed between using your own proxies and using them with Crawlera. This is because there is some overhead when routing requests through Crawlera as the request has to go through the Crawlera endpoint before going through the proxy.
+
+Crawlera also throttles requests and introduces delays to avoid being banned on the target website. These delays can differ depending on the target domain, as some popular sites have more rigorous anti-scraping measures than others.
```
It's not "throttles and introduces delays" but rather "it throttles by introducing delays".
Also it's not only about antiban measures - e.g. rate limit could serve as a way against bringing down weak webservers.

qrilka commented:
@rdowinton also we need to note that retries could add another reason for extra delays in Crawlera.

pablohoffman commented on crawlera.rst:
```diff
@@ -453,3 +453,10 @@ Where can I monitor my Crawlera usage?
+Why are requests slower through Crawlera as opposed to using proxies directly?
+------------------------------------------------------------------------------
+
+If you're using your own proxies, you may notice a discrepancy in speed between using your own proxies and using them with Crawlera. This is because there is some overhead when routing requests through Crawlera as the request has to go through the Crawlera endpoint before going through the proxy.
```
I would not mention the crawlera master overhead, or at least mention it less prominently than the thortlling (now it's mentioned first).

rdowinton commented:
Updated PR. Removed mention of Crawlera master overhead and described how throttling will also help prevent inadvertently taking down the target website.
